src/specs/networking.md

```diff
@@ -66,3 +68,26 @@ Accepting a `WireTxPayForMessage` into the mempool requires different logic than
 Transaction senders who want to pay for a message will create a [SignedTransactionDataPayForMessage](./data_structures.md#signedtransactiondatapayformessage) object, `stx`, filling in the `stx.messageShareCommitment` field [based on the non-interactive default rules](../rationale/message_block_layout.md#non-interactive-default-rules) for `k = AVAILABLE_DATA_ORIGINAL_SQUARE_MAX`, then signing it to get a [transaction](./data_structures.md#transaction) `tx`. This process is repeated with successively smaller `k`s, decreasing by powers of 2 until `k * k <= stx.messageSize`. At that point, there would be insufficient shares to include both the message and transaction. Using the rest of the signed transaction data along with the pairs of `(tx.signedTransactionData.messageShareCommitment, tx.signature)`, a `WireTxPayForMessage` object is constructed.
 
 Receiving a `WireTxPayForMessage` object from the network follows the reverse process: for each `message_commitment_and_signature`, verify using the [based on the non-interactive default rules](../rationale/message_block_layout.md#non-interactive-default-rules) that the signature is valid.
+
+## BadEncodingFraudProof
+
+| name          | type                                                                                        | description                                                                                                          |
+|---------------|---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `shareProofs` | [ShareProof](./data_structures.md#shareproof)`[]`                                           | The available shares in the offending row or column.                                                                |
+| `root`        | [HashDigest](./data_structures.md#hashdigest)                                               | The Merkle root of the offending row or column.                                                                      |
+| `proof`       | [NamespaceMerkleTreeInclusionProof](./data_structures.md#namespacemerkletreeinclusionproof) | The Merkle proof of the row or column root in [`availableDataRoot`](./data_structures.md#header).|
```

Comment:
The `availableDataRoot` uses the Merkle root (not namespace) of the compact row/col roots: https://github.com/celestiaorg/celestia-specs/blob/ec98170398dfc6394423ee79b00b71038879e211/src/specs/data_structures.md#availabledataheader

Reply:
Fixed.